.env

```diff
@@ -0,0 +1,7 @@
+ENGINE=mysql.connector.django
+NAME=app_db
+USER=app_user
+PASSWORD=1234
+HOST=mysql
+PORT=3306
+PYTHONUNBUFFERED=1
```

Comment:
Add EOF, it is needed to define the end of the file. To add EOF add one empty line at the end of the file
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_4_externalize_configs/assets/87720583/393deb98-c653-4456-bf4d-1506814827a7)